python/setup.py

```diff
@@ -13,9 +13,9 @@
     name="lib_xud",
     packages=setuptools.find_packages(),
     install_requires=[
-        "black~=21.5b1",
-        "pytest~=6.2",
-        "pytest-xdist~=2.3",
+        "black",
+        "pytest",
+        "pytest-xdist",
```

Comment:
As for requirements.txt, I think these should be pinned.

Reply:
are we actually using black?

Reply:
Not as far as I know.